README.md

```diff
@@ -52,7 +52,35 @@ In order to run this project you need: only to clone this github repo
 Clone this repository to your desired folder:
 
 - cd my-folder
-- git clone 'git@github.com:husush23/capstone--2.git'
+- git clone 'https://github.com/husush23/Math-magician.git'
+
+### Tech stack
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Additional features:
+
+- components
+- Calculator design
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to move these sections outside of the `getting started section, you can add this before this section ☝🏼 as suggested above
```suggestion

```